### PR TITLE
update module commands with new format

### PR DESCRIPTION
**Pull request by KevinMenden**
This updates the module commands `create` and `create-test-yml` with the new module format for the `test.yml` 

Instead of `tool_subtool` the tag will now be `tool/subtool`  (see here: https://github.com/nf-core/modules/pull/389) 

Still need to update the lint command to look for the correct tags if we decided to go only for these tags instead of having things like "my_tool_single_end" etc.

### CODE REVIEW

File: nf_core/modules/test_yml_builder.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def build_single_test(self, entry_point):
             mod_name_parts = self.module_name.split("/")
             tag_defaults = []
             for idx in range(0, len(mod_name_parts)):
-                tag_defaults.append("_".join(mod_name_parts[: idx + 1]))
+                tag_defaults.append("/".join(mod_name_parts[: idx + 1]))
```
Comment (drpatelh) [inline nf_core/modules/test_yml_builder.py:173]: So this will change the `tag` in `test.yml`?

Comment (drpatelh) [reply nf_core/modules/test_yml_builder.py:173]: To `bwa/index` instead of `bwa_index`?

**Comment by drpatelh** [comment]
Thanks @KevinMenden ! Need to change the format of the module in `pytest_software.yml` too when creating modules.

File: nf_core/modules/test_yml_builder.py (hunk 1 above)
Comment (KevinMenden) [reply nf_core/modules/test_yml_builder.py:173]: yep, that should do the trick.

**Comment by KevinMenden** [comment]
Right, thanks @drpatelh ! Changed that, now it should work.

File: nf_core/modules/lint.py [hunk 2]
```diff
@@ -519,6 +519,7 @@ def lint_module_tests(self):
         # Lint the test.yml file
         try:
             with open(self.test_yml, "r") as fh:
+                # TODO: verify that the tags are correct
```
Comment (drpatelh) [inline nf_core/modules/lint.py:522]: Shall we leave this for another PR?

**Comment by KevinMenden** [comment]
`pytest_software.yml` now looks like this:

```
tool/subtool:
  - software/tool/subtool/**
  - tests/software/tool/subtool/**

```

File: nf_core/modules/lint.py (hunk 2 above)
Comment (KevinMenden) [reply nf_core/modules/lint.py:522]: yeah I would say, nothing I want to do too quickly

**Review by drpatelh — APPROVED** [review]
LGTM!
